exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/access_store_ui.ipynb

```diff
@@ -10,6 +10,31 @@
     "<b>This notebook is not supposed to be used on its own.<b>"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "74f55112-92dc-4a54-ac1a-b068fadfbcc7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "\n",
+    "def upward_file_search(file_name: str) -> str:\n",
+    "\n",
+    "    dir = Path().resolve()\n",
```

Comment:
Could we move this functionality to notebook-connector to enable to apply pytest test cases?

Reply:
Yes, it's a good idea.